resources/sql/queries.sql

```diff
@@ -145,7 +145,8 @@ RETURNING repo_id, issue_id, issue_number, title, commit_sha, contract_address;
 -- :name update-transaction-hash :! :n
 -- :doc updates transaction-hash for a given issue
 UPDATE issues
-SET transaction_hash = :transaction_hash
+SET transaction_hash = :transaction_hash,
+    updated = timezone('utc'::text, now())
```

Comment:
Agree with introducing separate columns for timestamps, but perhaps not in this PR. It would be perhaps nice to have a separate table, something like `tx_data(hash, type, updated, issue_id)` and link it to `issues`. What do you think?

Reply:
Let's capture that in an issue so we don't forget

Reply:
Created a new issue for this: https://github.com/status-im/open-bounty/issues/442